frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -927,12 +927,8 @@ function ReaderHighlight:onHoldPan(_, ges)
         -- With CreDocuments, allow text selection across multiple pages
         -- by (temporarily) switching to scroll mode when panning to the
         -- top left or bottom right corners.
-        local mirrored_reading = BD.mirroredUILayout()
-        if self.ui.rolling and self.ui.rolling.inverse_reading_order then
-            mirrored_reading = not mirrored_reading
-        end
         local is_in_prev_page_corner, is_in_next_page_corner
-        if mirrored_reading then
+        if self.view.inverse_reading_order ~= BD.mirroredUILayout() then
```

Comment:
I'd rather still see the logic more explicite, with the name "mirrored_reading" - your one line optimization makes it less obvious to grasp what this does.
```lua
local mirrored_reading = BD.mirroredUILayout()
if self.view.inverse_reading_order then mirrored_reading  = not mirrored_reading end
if mirrored_reading then
```

Reply:
It's so simple and clear!

Reply:
Would a small comment to the line be enough?

Reply:
> It's so simple and clear!

For you who have had your nose in this for months now :), maybe.
But even for me who introduced BD.mirroredUILayout() 2 years ago, I had to stop on this to understand how the `~=` would work in the various combinations and why. If I had seen `mirrored_reading` written, I would not have stopped and just go on reading.